src/release/release.ts

```diff
@@ -145,6 +145,14 @@ export interface ReleaseProjectOptions {
    * @default - no prefix
    */
   readonly releaseTagPrefix?: string;
+
+  /**
+   * Custom configuration used when creating changelog with standard-version package.
+   * Given values either append to default configuration or overwrite values in it.
+   *
+   * @default - standard configuration applicable for GitHub repositories
+   */
+  readonly versionrc?: object;
```

Comment:
I think based on the naming scheme we use in other projen APIs, `versionrcOptions` would be a more fitting name here (also leaves open the possibility of `versionrc` in the future to be a boolean enable/disable property, or a string property to rename the file, etc.). Also I think we can make the type slightly more specific so the API is friendlier in other JSII languages:
```suggestion
  readonly versionrcOptions?: Record<string, any>;
```

Reply:
Made change through the PR to use `versionrcOptions` (or one in capital letters)